lib/hedwig_mopidy/responders/mopidy.ex

```diff
@@ -199,14 +208,40 @@ defmodule HedwigMopidy.Responders.Mopidy do
     send message, response
   end
 
-  hear ~r/^dj\svolume$/i, message do
+  hear ~r/^dj\s(skip|next)$/i, message do
+    currently_playing = HedwigMopidy.currently_playing
+    current_playlist = CurrentPlaylistStore.retrieve
+    response =
+      with {:ok, %TlTrack{} = next_track} <- Tracklist.next_track,
+           {:ok, :success} <- Playback.next do
+            HedwigMopidy.notice_message("Skipping to next track: #{HedwigMopidy.playing_string(next_track, current_playlist)}")
+      else
+        {:error, error_message} -> error_message
+        _ -> HedwigMopidy.notice_message("No more songs are queued")
+      end
+    send message, response
+  end
+
+  hear ~r/^dj\s(what|who).*\snext$/i, message do
```

Comment:
I've added help for this command. As far as the regex being too generic, I've reused what's already there for what/who's playing. We can create a new issue for being more specific in both commands if desired